### PR TITLE
chore: add exec_seconds attribute to ExecutionMetrics

### PR DESCRIPTION
**Pull request by Genesis929**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: bigframes/session/metrics.py [hunk 1]
```diff
@@ -64,13 +66,12 @@ def get_performance_stats(
...
 
-    exec_seconds = (
-        (query_job.ended - query_job.created).total_seconds()
-        if query_job.created is not None and query_job.ended is not None
-        else None
-    )
+    if query_job.created is not None and query_job.ended is not None:
+        execution_secs = (query_job.ended - query_job.created).total_seconds()
+    else:
+        return None
```
Comment (shobsi) [inline bigframes/session/metrics.py:72]: there are multiple `return None` statements which makes it a bit hard to follow. Can we do it only once in the beginning `if query_job.state != "DONE"`, otherwise always return a legit tuple?

Comment (Genesis929) [reply bigframes/session/metrics.py:72]: Seems query_job.state != "DONE" does not guarantee the metrics exist for case like mocks and dry_run, so maybe this is more reliable.

**Review by shobsi — APPROVED** [review]
